app/src/main/res/values/strings.xml

```diff
@@ -84,6 +84,7 @@
   <string name="leave_comment">Leave a comment…</string>
   <string name="only_backers_can_leave_comments">Only backers can leave comments</string>
   <!-- comment card -->
+  <string name="something_went_wrong">Something went wrong. Pull \ndown to reload this page.</string>
```

Comment:
I think there should already be a translated string in place already, as the translation pull request was merged this morning

Reply:
forget about this one, it seems still some translations not ready yet